onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -51,6 +51,14 @@ bool NodeGroupSelector::CheckQDQNodes(const GraphViewer& graph_viewer, const Nod
     return false;
   }
 
+  auto does_node_produce_graph_output = [&graph_viewer](const Node* node_ptr) {
```

Comment:
Is this issue better addressed by duplicating the DQ node so it doesn't have multiple consumers (if we consider the graph output to be a consumer)?  If so we can move the script that does that duplicate to be an L1 transformer and update it to consider a graph output to be a consumer.

Reply:
This is a hot fix for the corner case. Let's the duplication solution in the next release.

Reply:
> This is a hot fix for the corner case. Let's the duplication solution in the next release.

Agreed - was just checking that would be the better fix.